backfill/backfill_manager.py

```
@@ -0,0 +1,474 @@
+        logger.info(f"STARTING BACKFILL")
+        logger.info(f"Directory: {petition_dir}")
+        logger.info(f"Years: {start_year} - {end_year or 'current'}")
+        logger.info(f"Chapter filter: {chapter_filter or 'All'}")
+        logger.info(f"Dry run: {self.dry_run}")
+        logger.info("=" * 80)
+        
+        # Organize files by year
+        organized_files = self.organize_petitions_by_year(petition_dir)
+        
+        # Filter years
+        current_year = datetime.now().year
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)